### PR TITLE
Test referrerpolicy with preload

### PR DESCRIPTION
**Pull request by noamr**
When <preload referrerpolicy=> is used:
- The referrerpolicy should be respected
- It does not affect consuming the preload (e.g. the consumer can have a different referrerpolicy)
- Should work for link headers and elements

### CODE REVIEW

File: preload/preload-referrer-policy.html [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+        return {unsafe: location.href, actualReferrer: window.referrers[id]}
+    },
+}
+function test_referrer_policy(preloadPolicy, resourcePolicy, crossOrigin, type) {
+    promise_test(async t => {
+        const id = token();
+        const href = `${crossOrigin ? REMOTE_ORIGIN : ''}/preload/resources/echo-referrer.py?uid=${id}`;
+        const {actualReferrer, unsafe} = await loaders[type](t, {preloadPolicy, resourcePolicy, id, href})
+        const origin = window.origin + '/';
+//        if (type === 'header')
```
Comment (yoavweiss) [inline preload/preload-referrer-policy.html:64]: leftovers

File: preload/preload-referrer-policy.html [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+        document.body.appendChild(script);
+        await loaded;
+        return {unsafe: location.href, actualReferrer: window.referrers[id]}
+    },
+}
+function test_referrer_policy(preloadPolicy, resourcePolicy, crossOrigin, type) {
+    promise_test(async t => {
+        const id = token();
+        const href = `${crossOrigin ? REMOTE_ORIGIN : ''}/preload/resources/echo-referrer.py?uid=${id}`;
+        const {actualReferrer, unsafe} = await loaders[type](t, {preloadPolicy, resourcePolicy, id, href})
```
Comment (yoavweiss) [inline preload/preload-referrer-policy.html:62]: You got the href and the ID inversed here. I wonder how this was still working..

Comment (noamr) [reply preload/preload-referrer-policy.html:62]: They're named (in a `{}`), the order doesn't matter. I'll flip it for readability though

Comment (yoavweiss) [reply preload/preload-referrer-policy.html]: Oops, I knew I was missing something.. :)

**Review by yoavweiss — APPROVED** [review]
LGTM

**Pull request merged**